### ISSUE TITLE
[Feature Request] Aura only applies if certain conditions or status effects are present

### ISSUE THREAD

**Issue by WThunderspirit** [kandashi/Active-Auras#300]
**Is your feature request related to a problem? Please describe.**
No, AA always works fine for me

**Describe the solution you'd like**
Both a formulaic and Status Effect related conditional added to the list of options that a target could be under the effects of: For Examples:
- ##attributes.hp.value < ##attributes.hp.max/2 (If the targets current hp is less than half its max; aura effects them, otherwise it doesn't)
- @attributes.hp.value < @attributes.hp.max/2 (If the my current hp is less than half my max; aura effects them, otherwise it doesn't)
- (From drop down box related to Status Effects listed, or Icons if they only have Icons:) "Frightened" (Effects only Frightened targets)

**Describe alternatives you've considered**
Can't think of any

**Additional context**
So in D&D 5e, the **Conquest Paladin** has the **Aura of Conquest**:
> Starting at 7th level, you constantly emanate a menacing aura while you're not incapacitated. The aura extends 10 feet from you in every direction, but not through total cover.
> If a creature is frightened of you, its speed is reduced to 0 while in the aura, and that creature takes psychic damage equal to half your paladin level if it starts its turn there.
> At 18th level, the range of this aura increases to 30 feet.

I tried everything I could think of to make that work from the frightened condition, but the status effects are impossible to find with Actor Attribute Lists, so I got nowhere with midi-qol; my first thought.  then I thought, but why not feed that data from conditionals within the aura itself?  This would work very similar to the 'type' of creatures effected, just having different parameters than type.  So in the "Describe the solution you like, the third example is what inspired me to ask for the feature.

Personally I use DFred's Convenient effects and Monk's Little Details.  CUB I only use to effect Token Status Size.  I am still on V10.303 as well, as my potato PC can't handle V11, and D&D Core is 2.4.0; if any of that helps.

**Comment by MrPrimate** [comment]
In the "Custom Check" condition, you could enter something like:

`actor.appliedEffects.some(e => e.statuses.has('fear'))`

**Comment by MrPrimate** [comment]
For the targets hp, try:

`actor.system.attributes.hp.value < (actor.system.attributes.hp.max/2)`

You can't currently target the effect origin actor this way for the second example.

**Comment by WThunderspirit** [comment]
> In the "Custom Check" condition, you could enter something like:
> 
> `actor.appliedEffects.some(e => e.statuses.has('fear'))`

I should have probably stated this previously, but I'm working with V10 B312 Foundry, on Forge, with D&D 2.3.1 - both V10.312 and D&D 2.3.1 hardcapped to not update (had issues with updating beyond this).

Also, sadly, I really can't code Javascript yet, but I tried putting your code in verabtim and it comes up as "undefined" - even in F12 console; not true or false like I think it should.  I also tried 'frightened', since that is what my status calls the condition.  Didn't work.  I wonder if there would be a way to test against the convenient effect rather than the status effect, since I use DFred's CE to apply status conditions?  That may be getting into a rabbit hole though.

Don't mean to put you out, but I kinda need the spoonfed version; however, if there were comments in code to describe how this works, that would help me wrap my head around the changes since Netscape Navigator and Internet Explorer were the only browsers out there, and AOL was in it's death spasms (a.k.a; the last time I learned Javascript, when it was totally DOM dependent and far more limited than it's complicated but robust code now.  Learning it now seems like when I tried to learn Perl and PHP back in the day; and I've forgotten more than I can remember of those days.  Still know my HTML 4.0 though!  LMAO!)
